apps/db/lib/db/resource.ex

```diff
@@ -547,4 +571,7 @@ defmodule DB.Resource do
       where: resource.id == ^id
     )
   end
+
```

Comment:
tu peux te passer d'une nouvelle fonction : 
```
> m = nil
> Map.merge(m || %{}, %{"validation" => "coucou"})
%{"validation" => "coucou"}
> Map.merge(m || %{}, %{"validation" => "coucou"})
%{"size" => 1, "validation" => "coucou"}
```

Reply:
Ah, merci ! J'avais essayé avec un `or`. `or` et `||` sont équivalents en Python

Reply:
ah oui en Elixir les `and` et `or` sont réservés aux vrais booléens, mais pas les `&&` et `||`